data/card_catalogue/card-manipulation.json

```diff
@@ -226,7 +200,7 @@
     "tier": "blue",
     "details": {
       "rules": {
-        "active": "Draw 3 greens, face-up. Cast up to 3 and discard the rest.",
```

Comment:
This is just a typo I presume. This effect is way stronger then everything else if you get to cast 3 of a higher rarity.